torch/_dynamo/variables/torch.py

```diff
@@ -241,7 +241,17 @@ def call_function(
         elif self.value in REWRITE_OPS_TO_TENSOR_SIZE_METHOD:
             assert len(args) == 1
             assert isinstance(args[0], TensorVariable)
-            return args[0].call_method(tx, "size", [], {})
+            size = args[0].call_method(tx, "size", [], {})
+            return wrap_fx_proxy(
+                tx=tx,
+                proxy=tx.output.create_proxy(
+                    "call_function",
+                    torch.tensor,
+                    *proxy_args_kwargs([size], {}),
+                    current_tx=tx,
+                ),
+                **options,
+            )
```

Comment:
```torch._shape_as_tensor``` should return ```Tensor``` rather than ```size```.

Reply:
The reason we return size is because we expect the most use cases to be:
`b0, b1 = torch.onnx.shape_as_tensor(x)`

I am afraid this will break above code.